tests/recordRequests.js

```diff
@@ -3,53 +3,72 @@ const fs = require(`node:fs`);
 const path = require(`node:path`);
 const v8 = require(`node:v8`);
 
-const nock = require(`nock`);
+/**
+ * @type {Map<string, {body: ArrayBuffer, status:number, headers: Record<string,string>}>}
+ */
+let mocks = new Map();
 
-const getNockFile = () =>
-  path.join(
+function getNockFile() {
+  return path.join(
```

Comment:
Consider keeping the old syntax to minimize the diff
```suggestion
const getNockFile = () =>
  path.join(
```